### PR TITLE
[OCPBUGS-14626]: Removing references to oc login from etcd restore doc

### PR DESCRIPTION
**Pull request by lahinson**
Version(s): 4.10+

Issue: https://issues.redhat.com/browse/OCPBUGS-14626

Link to docs preview: https://62131--docspreview.netlify.app/openshift-enterprise/latest/backup_and_restore/control_plane_backup_and_restore/disaster_recovery/scenario-2-restoring-cluster-state.html#dr-scenario-2-restoring-cluster-state_dr-restoring-cluster-state

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by palonsoro** [comment]
@lahinson I am afraid that step 17 makes no sense as it is now. The command placed is simply wrong.

Step 17 should be replaced with either of these options:
- In a separate terminal, please export `KUBECONFIG` variable to point to a cluster-admin certificate based kubeconfig.
- Open a separate terminal in the recovery host,  export the recovery kubeconfig by running `export KUBECONFIG=/etc/kubernetes/static-pod-resources/kube-apiserver-certs/secrets/node-kubeconfigs/localhost-recovery.kubeconfig`

And I have now realized: The step 18 wouldn't make any sense with the step 17 unless the sentence "In **a** terminal that has access to the cluster as a cluster-admin user" is replaced by "In **the** terminal that has access to the cluster as a cluster-admin user". Basically because you don't want to do step 18 in any terminal but in the one that was prepared as per step 17.

**Comment by lahinson** [comment]
@palonsoro Thank you for the information! I updated steps 17 and 18 according to your feedback. I used the more guided option that you presented. Let me know what you think when you can.

**Comment by palonsoro** [comment]
It looks now good to me!

/lgtm

**Comment by lahinson** [comment]
Hi @geliu2016 - When you have some time, PTAL at this PR. Thank you!

**Review by cbippley — COMMENTED** [review]
LGTM!

**Comment by cbippley** [comment]
@opayne1 Want to take a look?
/label peer-review-done
/remove-label peer-review-in-progress

**Review by opayne1 — COMMENTED** [review]
Just a few comments on my end!

File: modules/dr-restoring-cluster-state.adoc [hunk 1]
```diff
@@ -22,7 +22,7 @@ When you restore your cluster, you must use an etcd backup that was taken from t
 
 .Prerequisites
 
-* Access to the cluster as a user with the `cluster-admin` role.
+* Access to the cluster as a user with the `cluster-admin` role through a certificate-based kubeconfig file, like the one that was used during installation.
```
[the remark this answers is not in the document]
Comment (lahinson) [reply modules/dr-restoring-cluster-state.adoc]: I added the backticks, but I think I'll leave "certificate-based" for now, as that's what the engineer specified.

**Pull request merged**